src/pkg/cluster/zarf.go

```diff
@@ -109,28 +109,29 @@ func (c *Cluster) StripZarfLabelsAndSecretsFromNamespaces(ctx context.Context) {
 	spinner.Success()
 }
 
-// PackageSecretNeedsWait checks if a package component has a running webhook that needs to be waited on.
-func (c *Cluster) PackageSecretNeedsWait(deployedPackage *types.DeployedPackage, component v1alpha1.ZarfComponent, skipWebhooks bool) (needsWait bool, waitSeconds int, hookName string) {
+// PackageSecretNeedsWait checks if a package component has a running webhook that needs to be waited on. Returns the
+// number of seconds remaining to wait and the name of the webhook. If seconds is zero there's no need to wait.
+func (c *Cluster) PackageSecretNeedsWait(deployedPackage *types.DeployedPackage, component v1alpha1.ZarfComponent, skipWebhooks bool) (int, string) {
```

Comment:
Probably worth just making a separate PR for this change, and keep this PR focused on removing named returns

Reply:
Yeah I definitely agree with changing the name on this, it was one path I went through before moving forward with this. For this specific case, the named returns are doing a lot of the documentation for the args are. I think this entire method is better suited for a future refactor rather than trying to fix it now, so I'm going to leave it as is and make an issue.

Reply:
tech debt issue: https://github.com/zarf-dev/zarf/issues/2986